examples/src/main/scala/example/stitching/ExampleApp.scala

```diff
@@ -0,0 +1,126 @@
+package example.stitching
+
+import zio._
+
+import caliban._
+import caliban.GraphQL.graphQL
+import caliban.schema._
+import caliban.tools.{ Options, RemoteSchema, SchemaLoader }
+import caliban.tools.stitching.{ HttpRequest, RemoteResolver, RemoteSchemaResolver, ResolveRequest }
+
+import sttp.client3.asynchttpclient.zio._
```

Comment:
minor: move this import above the zio one

Reply:
Fixed in bc5c35d. I also moved both of them below the caliban imports.